gradle/libs.versions.toml

```diff
@@ -0,0 +1,64 @@
+[versions]
+agp = "7.1.0"
+autoService = "1.0.1"
+autoValue = "1.10"
+dagger = "2.39.1"
+espresso = "3.5.1"
+kotlin = "1.8.0"
+ktlint = "0.41.0"
+
+# Configs that we can override in CI. Not exactly "versions" but useful to repurpose it here.
+config-fullTestRun = "true"
+config-generateDaggerFactoriesWithAnvil = "true"
+config-useIr = "true"
+
+[plugins]
+agp-application = { id = "com.android.application", version.ref = "agp" }
+agp-library = { id = "com.android.library", version.ref = "agp" }
+anvil = { id = "com.squareup.anvil", version = "2.4.4" }
```

Comment:
Unfortunately yes, gradle doesn't let you have a version-less plugin declaration even if it's unused

Reply:
I take it that it's not possible to set it dynamically with a function like we were previously, correct? Perhaps it would make sense to move the definition of `VERSION_NAME` to this file to keep that value being defined on only one place even if it doesn't technically make sense as part of `libs`. Just trying to think of what other options we have here.

Reply:
I'm not sure I understand, the version here does not matter as we replace it with dependency substitution in the build itself. The toml format just needs to have one set for its own sanity

Reply:
[d4a3ea3](https://github.com/square/anvil/pull/678/commits/d4a3ea335c6067b0bf7cc5eb04382fe71820b43f)